src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactElement, ReactNode } from 'react';
+import * as Sentry from '@sentry/react';
+import { ApiErrorException } from '@/api/errors';
+import styled from 'styled-components';
+import AlertStripe from 'nav-frontend-alertstriper';
+
+export enum ErrorContext {
```

Comment:
Kan vurdere å endre til `export type ErrorContext = 'mainPage' | 'appRouter' |'oversiktContainer'` for litt mindre kode ;)
Samme i NotificationType.

Reply:
Enig. Endrer!